src/components/about/ValueSection.tsx

```diff
@@ -1,3 +1,5 @@
+"use client";
+import { motion } from "framer-motion";
```

Comment:
import should be from motion/react
```suggestion
import { motion } from "motion/react";
```